lib/util/get-typescript-extension-map.js

```diff
@@ -40,7 +40,9 @@ function normalise(typescriptExtensionMap) {
         if (!typescript) {
             continue
         }
-        backward[javascript] ??= []
+        if (backward['javascript'] === null || backward['javascript'] === undefined) {
+            backward['javascript'] = [];
+        }
```

Comment:
This would also break the mapping, it should be `backward[javascript]`, like this:
```suggestion
        if (backward[javascript] === null || backward[javascript] === undefined) {
            backward[javascript] = [];
        }
```

Also, I don't remember if we support `== null`, but this would work too
```suggestion
        if (backward[javascript] == null) {
            backward[javascript] = [];
        }
```